# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 3bf656ec4a64
# Full Hash: 3bf656ec4a648238bfbcf41f0b446464af1c102d
# Author: Sotaro Ikeda <sotaro.ikeda.g@gmail.com>
# Date: 2025-06-12 21:42:30
# Regressor Bug: 1945683
# File Overlap Count: 1
# Description:
#   Bug 1945683 - Enable layer compositor on Windows in Nightly r=gfx-reviewers,lsalzman
#   
#   All known rendering bugs of layer compositor have been addressed. Then is seems OK to remove nightly only limitation of pref gfx.webrender.layer-compositor.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251788
# ==============================================================================

diff -r 6d252b985288 -r 3bf656ec4a64 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Thu Jun 12 20:50:42 2025 +0300
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Jun 10 11:15:01 2025 +0000
@@ -7530,14 +7530,14 @@
 #endif
   mirror: once
 
-#ifdef NIGHTLY_BUILD
-  # Keep this pref hidden on non-nightly builds to avoid people accidentally
-  # turning it on.
 - name: gfx.webrender.layer-compositor
   type: bool
-  value: false
-  mirror: once
-#endif
+#if defined(XP_WIN)
+  value: @IS_NIGHTLY_BUILD@
+#else
+  value: false
+#endif
+  mirror: once
 
 #ifdef XP_WIN
 - name: gfx.webrender.layer-compositor-use-composition-surface
